### PR TITLE
feat: add sync other npm script

### PR DESCRIPTION
**Pull request by any1024**
ref: #17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +63,7 @@
     "cross-env": "^7.0.3",
     "dayjs": "^1.11.9",
     "dumi": "^2.2.1",
+    "execa": "^5.1.1",
```
Comment (any1024) [inline package.json:66]: Execa 5. * Use `require` get  pkg , but execa latest 7. * Use`import` .
 `execa@7. *` Unable to run successfully under ts-node, I tried to modify some configurations. But it has never been successful.

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "site:build": "dumi build",
     "site:deploy": "gh-pages -d site",
     "start": "npm run dev",
+    "sync:other-npm": "node -e 'require(\"./scripts/sync-tnpm-with-cnpm\")()'",
```
Comment (dengfuping) [inline package.json:21]: - `sync-cnpm-with-tnpm` => `sync-cnpm-and-tnpm`
- Add `postpublish` script to execute `sync:other-npm`

**Review by dengfuping — APPROVED** [review]

**Review by dengfuping — CHANGES_REQUESTED** [review]
Request changes

**Review by dengfuping — CHANGES_REQUESTED** [review]
- should fix conflicts

**Comment by any1024** [comment]
@dengfuping Help me see the reason for Preview Start Error

**Comment by dengfuping** [comment]
> @dengfuping Help me see the reason for Preview Start Fail

- I will fix this problem later. This PR could be merged.

**Pull request merged**